pkg/gimletd/notifications/podStateManager.go

```diff
@@ -0,0 +1,90 @@
+package notifications
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	gimletdConfig "github.com/gimlet-io/gimlet-cli/cmd/gimletd/config"
+	"github.com/gimlet-io/gimlet-cli/pkg/dashboard/api"
+	"github.com/gimlet-io/gimlet-cli/pkg/gimletd/model"
+	"github.com/gimlet-io/gimlet-cli/pkg/gimletd/store"
+	"github.com/sirupsen/logrus"
+)
+
+type podStateManager struct {
+	NotifManager Manager
+	WaitTime     int
+}
+
+func NewPodStateManager(notifManager Manager, waitTime int) *podStateManager {
+	return &podStateManager{NotifManager: notifManager, WaitTime: waitTime}
+}
+
+func (p podStateManager) Start(pods []api.Pod) {
+	gimletdConfig, err := gimletdConfig.Environ()
```

Comment:
Config should come from the entry point of the app.

As a param to this service.